sdks/wasm/Mono.WebAssembly.DebuggerProxy/DevToolsHelper.cs

```diff
@@ -190,8 +190,11 @@ public static MonoCommands ClearAllBreakpoints ()
 		public static MonoCommands GetDetails (DotnetObjectId objectId, JToken args = null)
 			=> new MonoCommands ($"MONO.mono_wasm_get_details ('{objectId}', {(args ?? "{}")})");
 
-		public static MonoCommands GetScopeVariables (int scopeId, params int[] vars)
-			=> new MonoCommands ($"MONO.mono_wasm_get_variables({scopeId}, [ {string.Join (",", vars)} ])");
+		public static MonoCommands GetScopeVariables (int scopeId, params VarInfo[] vars)
+		{
+			var var_ids = vars.Select (v => $"{{ index: {v.Index}, name: '{v.Name}' }}").ToArray ();
+			return new MonoCommands ($"MONO.mono_wasm_get_variables({scopeId}, [ {string.Join (",", var_ids)} ])");
```

Comment:
var_ids is already an array, should we serialize it for real?
```suggestion
			return new MonoCommands ($"MONO.mono_wasm_get_variables({scopeId},{JsonConvert.SerializeObject(var_ids)}");
```